### PR TITLE
Adding areas 85 introduced with 2.4 patch

### PR DESCRIPTION
**Pull request by Wyrda65**
* Adding new areas 85 introduced in patch 2.4
* Adding upping of Set Items

Immunities data is taken from https://www.reddit.com/r/Diablo/comments/s9w26y/overview_of_immunities_in_the_new_level_85_zones/
Mob count is taken from older D2 level configuration file https://github.com/fabd/diablo2/blob/17d60a1085151a640b164e047b4e0afb179af254/code/d2_113_data/Levels.txt, which might be changed. Indeed few areas had more packs when hence updated accordingly

### CODE REVIEW

**Comment by Wyrda65** [comment]
I've also added Set Upping

File: index.html [hunk 1]
```diff
@@ -5948,25 +5974,25 @@
 </table><br></td><td class='vatop'><table class='head' id='table.alvl3'>
     <tr class='gt gb'><th><a href='http://classic.battle.net/diablo2exp/maps/act3.shtml'><div class='wp'></div></a></th><th>III Kurast Docs          N  nm   H <span class='l'>Left</span>/<span class='c'>S</span>/<span class='r'>Right</span>  <button value='table.alvl3' onclick='onCollapse(this);'>-</button></th></tr>
     <tr><td><div class='wp'></div></td><td class='tooltip g11'>Spider Forest           21  49  <meta              />79<meta/> <span class='l'>    </span     ><span class='c'  >        </span   ><span class='r'>     </span></td></tr>
-    <tr><td                      ></td><td class='tooltip g22'>  Arachnid Lair         21  50  <meta              />79<meta/> <span class='l'>    </span     ><span class='c'  >        </span   ><span class='r'>     </span></td></tr>
+    <tr><td                      ></td><td class='tooltip g22'>  Arachnid Lair         21  50  <span class='alvl83'>85</span> <span class='l'>    </span     ><span class='c'  >        </span   ><span class='r'>     </span></td></tr>
```
Comment (JSterling8) [inline index.html:5977]: `alvl83` should be `alvl85`

**Review by JSterling8 — CHANGES_REQUESTED** [review]
Stony Tomb Level 1 and Level 2 are now level 85 areas but are still listed as level 78/79 in the area list at the bottom of the page around line #5950 or so.

File: index.html (hunk 1 above)
Comment (Wyrda65) [reply index.html]: fixed

**Comment by Michaelangel007** [comment]
Looks good.  Thanks for the patch!

**Pull request merged**